demo/clean-kubernetes.sh

```diff
@@ -6,9 +6,11 @@ set -aueo pipefail
 source .env
 
 for ns in "$BOOKWAREHOUSE_NAMESPACE" "$BOOKBUYER_NAMESPACE" "$BOOKSTORE_NAMESPACE" "$BOOKTHIEF_NAMESPACE" "$K8S_NAMESPACE"; do
-    kubectl delete namespace "$ns" --ignore-not-found
+    kubectl delete namespace "$ns" --ignore-not-found &
```

Comment:
We should just delete the namespaces in parallel:
`kubectl delete ns1 ns1 ...`

Reply:
this works serially on the back apparently: https://stackoverflow.com/questions/43675231/kubernetes-delete-all-jobs-in-bulk
I've found a wait to wait for resource to be truly deleted, kubectl `--wait` will do the job
https://kubernetes.io/docs/reference/generated/kubectl/kubectl-commands

Reply:
Where does it say its serial? Let's skip the wait for now, probably going to slow it down further.

Reply:
Delete using visitor pattern
https://github.com/kubernetes/kubectl/blob/master/pkg/cmd/delete/delete.go#L293
Reading down it still looks serial to me, the api doesn't say though. Seeing the visitor callback uses append to a slice in the outer scope, i'd say can't be parallel